hw/ip/spi_host/data/spi_host_testplan.hjson

```diff
@@ -66,68 +61,140 @@
                 once the event interrupt pin is asserted
             '''
       milestone: V2
-      tests: ["spi_host_error_event_intr"]
+      tests: []
     }
     {
-      name: tx_overflow_error
+      name: clock_rate
       desc: '''
-            Test TX OVERFLOW error is asserted by the spi_host
+            Stimulus:
+              - assert CONFIGOPTS_0.CSAAT_0 to combine messages
+              - select different settings for:
+              * CONFIGOPTS_0.CSNIDLE_0
+              * CONFIGOPTS_0.CSNLEAD_0
+              * CONFIGOPTS_0.CSNTRAIL_0
 
+            Checking:
+              - verify that merging of commands work correctly
+              - verify that the DUT can handle different sck -> cs_n timings
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: speed
+      desc: '''
+            Stimulus:
+              - randomly select the DUT to run single/dual/quad mode
+
+            Checking:
+              - verify that all speeds are supported
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_select_timing
+      desc: '''
             Stimulus:
-              - Program ERROR_ENABLE register to enable the TXOVERFLOW interrupt
-              - Program transaction with proper constraints to assert the TXOVERFLOW interrupts
+              - Randomly select a setting for the 16bit clock divider
 
             Checking:
-              - Ensure excess tx data is dropped
-              - Ensure the matching between the value of OVERFLOW bit in the ERROR_STATUS
-                and the ERROR_ENABLE register the once error interrupt pin is asserted
+              - Check that the DUT operates correctly under different SPI clock speeds
             '''
       milestone: V2
-      tests: ["spi_host_tx_overflow_error"]
+      tests: []
     }
     {
       name: component_reset
       desc: '''
             Test components (spi_fsm, rx_fifo, tx_fifo) are randomly reset
 
             Stimulus:
-              - Reset the components of spi_host randomly after a random number of data shows up on fifos
+              - Reset the components of spi_host randomly
+                after a random number of data shows up on fifos
 
             Checking:
               - Ensure that reads to RXDATA register yield 0s after the rx_fifo is reset
               - Ensure that transactions are dropped in both the scoreboard and spi_agent monitor
                 after the tx_fifo or spi_fsm is reset
             '''
       milestone: V2
-      tests: ["spi_host_component_reset"]
+      tests: []
     }
     {
-      name: clock_domain
+      name: modes
       desc: '''
-            TBD - Verify the function of clock-crossing domain for spi_host
+            - Verify the function of spi_host in different special_modes
+              (pass-through and manual_ctrl_cs)
```

Comment:
Manual CS control has been removed from the spec.

Reply:
Thank I have removed it